src/app/work-item/work-item-list/work-item-list-entry.component.html

```diff
@@ -0,0 +1,54 @@
+<div *ngIf="showDialog">	
+	<pf-dialog
+		(pfDialogClick)="onButtonClick($event)"
+		[dialog]='dialog'	
+	></pf-dialog>
+</div>
+<div class="list-group-item" (click)="onSelect($event)" [class.selected]="isSelected()">
+	<!-- info area -->
+	<div class="list-view-pf-main-info">
+        <div class="list-view-pf-left mobdn">
+            <pf-dropdown
+                (change)="onChangeState($event)" 
+                [options]="stateDropdownOptions"
+                [selected]="workItem.selectedState">
+            </pf-dropdown>				
+        </div>
+		<div class="list-view-pf-left type workItemList_workItemType">
+			<span class='fa' [ngClass]="{'fa-wrench': workItem.type=='system.bug',
+                                        'fa-user': workItem.type=='system.userstory',
```

Comment:
Fixed. It looks differently in my IDE. I think this is caused by tab/space behaviour.